execution/executors/shim/process.go

```diff
@@ -123,6 +130,7 @@ func loadProcess(root, id string) (*process, error) {
 	}()
 
 	p := &process{
+		// ctx is not set here (should we set context.TODO?)
```

Comment:
updated PR to pass ctx to loadProcess.

`log.G(nil)` should be another PR (I'm reconsidering if it is the right way)